### DISCUSSION TITLE
Why does this (and similar) plugin exist?

### DISCUSSION THREAD

**Discussion by windowsrefund** [LintaoAmons/cd-project.nvim#33] [General]
> I found all I need is an easier way to cd to another project directory.

Yep. But why solve for that in an editor? Seems to me,  you'd scope your project via a tmux session (or similar). 1 terminal, 1 multiplexer running multiple sessions where you can do anything (including using your editor).

**Comment by LintaoAmons** [comment]
I used to use tmux~

But it's missing something I need like fzf project and switch to it~

**Comment by LintaoAmons** [comment]
Yeah, I took another look, found there's `https://github.com/jrmoulton/tmux-sessionizer` which works really well, I'm going to switch back to tmux
